client/api/analytics/AnalyticsManager.cs

```diff
@@ -51,7 +51,7 @@ public void Start()
                 timer.Start();
                 
                 // SeenTargetsCache timer
-                seenTargetsCacheResetTimer = new Timer(config.seenTargetsCacheTtlInSeconds);
+                seenTargetsCacheResetTimer = new Timer((long)config.seenTargetsCacheTtlInSeconds * 1000);
```

Comment:
Unrelated but important fix for the seen targets cache not using the correct interval.